ibis/backends/duckdb/compiler.py

```diff
@@ -16,6 +21,46 @@ class DuckDBSQLExprTranslator(AlchemyExprTranslator):
     _has_reduction_filter_syntax = True
 
 
+@compiles(dt.UInt64, "duckdb")
+def compile_uint64(element, compiler, **kw):
+    return "UBIGINT"
+
+
+@compiles(dt.UInt32, "duckdb")
+def compile_uint32(element, compiler, **kw):
+    return "UINTEGER"
+
+
+@compiles(dt.UInt16, "duckdb")
+def compile_uint16(element, compiler, **kw):
+    return "USMALLINT"
+
+
+@compiles(dt.UInt8, "duckdb")
+def compile_uint8(element, compiler, **kw):
+    return "UTINYINT"
```

Comment:
The `@compiles` decorator lets us specialize how our custom unsigned sqlalchemy types compile for a specific dialect.

Reply:
Nice. The `@compiles` decorator is pretty neat.